nsdb-cluster/src/main/resources/application-k8s-dns.conf

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018 Radicalbit S.r.l.
```

Comment:
Maybe it's a stupid question....do these configs need var env also?

Reply:
It's anything but stupid. 
Basically, we do not need to change ports defined in this file, because those are ports used internally by the k8s cluster and not exposed to the users. 
At an infrastructure level, the exposed ports that mapped the ones defined here are configured.